magento/pwa-studio
# [PWA-893] My Account: Order History - Remove @client directive

## Description

This feature is being included in PWA v8, and to support 2.4.0, it was developed with mock data and uses the @client directive so it doesn't hit the actual backend for data. Once all of our backing instances have been upgraded to 2.4.1, our support matrix for the next release (v9) will shift to 2.4.1-2.4.2, and this @client directive can safely be removed.

This should go smoothly, but a full round of QA will need to happen now that Devagouda Patil can test against live data.

Acceptance Criteria
@client directive removed
MOCK_ORDERS and CUSTOM_TYPES are removed, and useTypePolices call in useOrderHistoryPage is removed
Configurable product images should load with real data.

## Related Issue





* [[PWA-893](https://jira.corp.magento.com/browse/PWA-893)] My Account: Order History - Remove @client directive
## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Login as Customer with Orders and go to Order History Page
2. Verify that everything continues rendering as it did before with mock data

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
